### PR TITLE
150 export csv

### PR DESCRIPTION
**Pull request by VaabelMaret**

### CODE REVIEW

**Comment by IngaSokman** [comment]
On the user management table the pagination has more pages than there are users.

File: src/pages/UsersManagement/UsersManagement.tsx [hunk 1]
```diff
@@ -1,32 +1,48 @@
...
         <Tooltip
           title={t('cheat_sheet.user_management.title')}
           modalContent={<UserManagementCheatSheet />}
         />
         <Button
-          // href="/settings/users/add"
           appearance={AppearanceTypes.Secondary}
-          className={classNames({ [classes.invisible]: isEmpty(users) })}
+          className={classNames({
+            [classes.invisible]: !includes(userPrivileges, 'EXPORT_USER'),
```
Comment (IngaSokman) [inline src/pages/UsersManagement/UsersManagement.tsx:42]: Let's use hidden prop for hiding the button.

Comment (VaabelMaret) [reply src/pages/UsersManagement/UsersManagement.tsx:42]: yes, hidden prop is usually the best option, but currently I wanted to make it invisible not remove it.

File: src/helpers/index.ts [hunk 2]
```diff
@@ -74,3 +79,19 @@ export const objectsToCsvFile = <ValuesType>(
...
   return file
 }
+
+export const downloadFile = ({
+  data,
+  fileName,
+  fileType,
+}: DownloadFileProps) => {
+  const blob = new Blob([data], { type: fileType })
+  console.log('blob', blob)
```
Comment (KaarelKa) [inline src/helpers/index.ts]: **[blocker]**

Lets remove console logs before merging as well as any other lingering test code

File: src/pages/UsersManagement/UsersManagement.tsx [hunk 3]
```diff
@@ -19,6 +20,12 @@ const UsersManagement: FC = () => {
     handlePaginationChange,
   } = useFetchUsers()
   const { t } = useTranslation()
+  const { userPrivileges } = useAuth()
+  const { downloadCSV, isLoading } = useDownloadUsers()
+
+  const handelDownloadFile = () => {
```
Comment (KaarelKa) [inline src/pages/UsersManagement/UsersManagement.tsx]: **[non-blocker]**

handle* :D

**Review by KaarelKa — CHANGES_REQUESTED** [review]

**Review by IngaSokman — APPROVED** [review]

**Pull request merged**